modules/installation/pages/install-server-unified.adoc

```diff
@@ -6,18 +6,20 @@
 This section describes how to install {susemgr} Server from the {sle} installation media.
 For this topic we assume that you already have valid organization credentials with {scc} and have obtained a registration code for your {susemgr}.
 
-For information on registering with {scc}, retrieving your organization credentials from {scc}, or obtaining installation media, see xref:general-requirements.adoc[].
+For information on registering with {scc}, retrieving your organization credentials from {scc}, or obtaining installation media, see xref:installation:general-requirements.adoc[].
 
-[NOTE]
-====
-To change a {sles}{nbsp}{sles-version}{nbsp}{sp-version} into a {susemgr}{nbsp}{productnumber}, follow the instructions at xref:installation:install-vm.adoc[].
-====
+Before installing {susemgr}, ensure your physical or virtual machine has enough disk space and RAM by checking the requirements at xref:installation:hardware-requirements.adoc[].
 
 [NOTE]
 ====
-Before installing {susemgr}, ensure your physical or virtual machine has enough disk space and RAM by checking the requirements at xref:installation:hardware-requirements.adoc[].
+* The recommended way of installing {susemgr} is from the {sle} installation media with the Unified Installer.
+* In case of installing {susemgr} in a public cloud where {susemgr} image is available, use that image. 
+    For more information, see xref:quickstart-public-cloud:qs-publiccloud-overview.adoc[].
+* In case of installing {susemgr} in a public cloud where {susemgr} image is not available, it is possible to change a {sles}{nbsp}{sles-version}{nbsp}{sp-version} into a {susemgr}{nbsp}{productnumber}.
```

Comment:
```suggestion
* In case of installing {susemgr} in a public cloud where a {susemgr} image is not available, it is possible to start from a {sles}{nbsp}{sles-version}{nbsp}{sp-version} and switch the base product to {susemgr}{nbsp}{productnumber}.
```